src/server/graphql/mutations/acceptTeamInviteNotification.js

```diff
@@ -44,10 +44,13 @@ export default {
     publish(NEW_AUTH_TOKEN, viewerId, UPDATED, {tms});
     auth0ManagementClient.users.updateAppMetadata({id: viewerId}, {tms});
 
+    // remove the old invitation
+    publish(INVITATION, teamId, AcceptTeamInviteNotificationPayload, data, subOptions);
+
     // Tell the new team member about the team, welcome them, and remove their outstanding invitation notifications
     publish(TEAM, viewerId, AcceptTeamInviteNotificationPayload, data, subOptions);
 
-    // Tell the rest of the team about the new team member, toast the event, and remove their old invitations
+    // Tell the rest of the team about the new team member, toast the event
     publish(TEAM_MEMBER, teamId, AcceptTeamInviteNotificationPayload, data, subOptions);
```

Comment:
Sure thing! I think I meant the server-side pub/sub channel, not the actual GraphQL subscription. Something like:

```
                                                       / TeamGraphQLSubscriptionIterator
                           (within the server proc)   /
NotificationPayload - - -> [ notification channel ] - - -> TeamMemberGraphQLSubscriptionIterator
                                                      \
                                                       \ ...GraphQLSubscriptionIterator
```

Just thinking out loud here about the relationship between pub/sub channels and GraphQL subscriptions, and how coupled they should or shouldn't be!

Reply:
my response would be very tightly coupled. a graphql subscription owns 1 or more channels. that channel is exclusive to the graphql sub.

if a graphql sub only owned only 1 channel, then we'd have to publish to every `userId` instead of `teamId`. Not the end of the world, but not great.

if 2 graphql subs shared a channel, then we'd have to bake in a bunch of filtering, so it'd be just like 2 channels, anyways. so, we could call `publish(CATCH_ALL)`, but then we'd need mutation-specific logic inside that catchall. i'd rather keep the logic in the resolve function.